sn_protocol/src/messages/query.rs

```diff
@@ -29,6 +29,13 @@ pub enum Query {
     /// [`Chunk`]:  crate::storage::Chunk
     /// [`GetChunk`]: super::QueryResponse::GetChunk
     GetChunk(ChunkAddress),
+    /// Retrieve a [`Register`] at the given address.
+    ///
+    /// This should eventually lead to a [`GetRegister`] response.
+    ///
+    /// [`Register`]: sn_registers::Register
```

Comment:
The response would be a `SignedRegister` instead of a `Register` here.
```suggestion
    /// Retrieve a [`SignedRegister`] at the given address.
    ///
    /// This should eventually lead to a [`GetRegister`] response.
    ///
    /// [`SignedRegister`]: sn_registers::SignedRegister
```